spring-kafka/src/main/java/org/springframework/kafka/listener/KafkaMessageListenerContainer.java

```diff
@@ -926,7 +926,6 @@ else if (listener instanceof MessageListener) {
 			if (this.containerProperties.isLogContainerConfig()) {
 				this.logger.info(toString());
 			}
-			Map<String, Object> props = KafkaMessageListenerContainer.this.consumerFactory.getConfigurationProperties();
```

Comment:
Why is this change?

Reply:
I obtained it from previous PR, but this field isn't used because `ErrorHandlingUtils.checkDeserializer` itself goes to `ConsumerFactory` to grab configuration properties. This was done in this [commit](https://github.com/spring-projects/spring-kafka/commit/e9564134733e81ba530d83193b68c0c34cb7a556). Perhaps it would be better to extract these changes into another refactoring PR?

Reply:
I suggest not including changes unrelated to the core features we are adding through the PR. Otherwise, it will be confusing if we have to re-review this code later.

Reply:
Fixed.